recipe/meta.yaml

```diff
@@ -31,15 +31,30 @@ test:
 
   imports:
     - psutil
+    - psutil._psutil_posix    # [unix]
+    - psutil._psutil_linux    # [linux]
+    - psutil._psutil_osx      # [osx]
+    - psutil._psutil_windows  # [win]
```

Comment:
We should be careful about testing private modules. Not a problem to test them. However if things change internally, we may need to drop them.